lib/rubocop/cop/solidus/class_eval_decorator.rb

```diff
@@ -3,28 +3,8 @@
 module RuboCop
   module Cop
     module Solidus
-      # TODO: Write cop description and example of bad / good code. For every
-      # `SupportedStyle` and unique configuration, there needs to be examples.
-      # Examples must have valid Ruby syntax. Do not use upticks.
-      #
-      #
-      # @example EnforcedStyle: SpreeClass
-      #   # Description of the `SpreeClass` style.
-      #
-      #   # bad
-      #   SpreeClass.class_eval do
-      #   .
-      #   .
-      #   end
-      #
-      #
-      #   # good
-      #   module SpreeClassDecorator
-      #   .
-      #   .
-      #   end
-      #
       class ClassEvalDecorator < Base
+        extend TargetRailsVersion
```

Comment:
Need to add module to cop to get required `minimum_supported_solidus_version?` method

(Doesn't work at the moment)

Reply:
Another option is to create our own base class that inherits from `Rubocop::Cop::Base` and add all our required functions there and modify our cops to inherit from that new base class.